api-docs/rst/dev-guide/api-operations/methods/put-update-policy-v1.0-tenantid-groups-groupid-policies-policyid.rst

```diff
@@ -218,20 +218,20 @@ This table shows the body parameters for the request:
 |                            |             |``schedule``.                              |
 +----------------------------+-------------+-------------------------------------------+
 |scalingPolicies.[*].\       |Integer      |The change to make in the number of        |
-|**change**                  |*(Optional)* |servers in the scaling group. This         |
+|**change**                  |*(Optional)* |units in the scaling group. This           |
 |                            |             |parameter must be an integer. If the value |
 |                            |             |is a positive integer, the number of       |
-|                            |             |servers increases. If the value is a       |
-|                            |             |negative integer, the number of servers    |
+|                            |             |units increases. If the value is a         |
+|                            |             |negative integer, the number of units      |
 |                            |             |decreases.                                 |
 +----------------------------+-------------+-------------------------------------------+
-|scalingPolicies.[*].\       |Integer      |The desired server capacity of the scaling |
-|**desiredCapacity**         |*(Optional)* |the group; that is, how many servers       |
+|scalingPolicies.[*].\       |Integer      |The desired scaling unit capacity of the   |
+|**desiredCapacity**         |*(Optional)* |scaling group; that is, how many units     |
 |                            |             |should be in the scaling group. This value |
```

Comment:
Delete..   The desired scaling unit ....   how many units should be in scaling group."

Replace with "Specifies the desired number of units to include in the scaling group."